packages/web/src/components/spa/Navigation/MobileNav/style.css

```diff
@@ -9,6 +9,31 @@
     color: white !important;
 }
 
+.navmobile-button-ham p {
```

Comment:
This is redundant. Add the font-size style in the `.navmobile-button-ham` class.